JSDemos/Demos/Validation/Overview/description.md

```diff
@@ -1,9 +1,34 @@
-DevExtreme Data Editors provide a&nbsp;powerful and seamless way to&nbsp;perform data validation. In&nbsp;this demo, editors are collected in&nbsp;an&nbsp;HTML&nbsp;form. Each editor is&nbsp;accompanied by&nbsp;the Validator component validating the editor against a&nbsp;set of&nbsp;specified rules. Editors are validated each time their value is&nbsp;changed. All validation errors are displayed in&nbsp;the ValidationSummary component.
+This demo shows how to implement a [default validation group](/Documentation/Guide/UI_Components/Common/UI_Widgets/Data_Validation/#Validate_Several_Editor_Values/Group_the_Editors) - a group of editors on the page with enabled data validation. In this demo, the editors are collected in an HTML form.
 
+To enable editor's data validation, you need to declare the [Validator](/Documentation/ApiReference/UI_Components/dxValidator/) component and implement the [validation rules](/Documentation/ApiReference/UI_Components/dxValidator/Validation_Rules/). You can attach several validation rules to one component. Each validation rule contains a rule type that must be one of the following:
 
+- [required](/Documentation/ApiReference/UI_Components/dxValidator/Validation_Rules/RequiredRule/)    
+A validation rule that demands that a validated field has a value.
```

Comment:
the [currently] validated field, that is the field to which the examined rule is applied. You did it right in the line below. Just be more attentive.

```suggestion
A validation rule that demands that the validated field has a value.
```